src/partials/article.hbs

```diff
@@ -6,6 +6,9 @@
 {{#with page.title}}
 <h1 class="page">{{{this}}}</h1>
 {{/with}}
+{{#if page.attributes.beta}}
```

Comment:
It might be better to also check that this page is a cloud doc so that we have the option to display the `Beta` tag for non-cloud docs.

Reply:
`page-cloud`

Reply:
done